client/lib/utils.js

```diff
@@ -142,6 +142,14 @@ Utils = {
                 return 'labels.' + labels[labelId] + '.' + key;
             }
         }
+    },
+
+    // Extract userId from user object
+    getUserId: function(user) {
```

Comment:
I think we don't want this function. In the code we should always know if a variable `x` is a user object (in this case we just use `x._id`) or an id (`x`). This function can bring confusion during the development.